tests/test_graph_types.py

```diff
@@ -113,14 +114,42 @@ def node_to_create(required_dep: int, optional_dep: int = 1) -> str:
     }
 
 
-def test_create_hamilton_node_missing_version():
+def test_create_hamilton_config_node_verssion():
+    """Config nodes now return the name as the version."""
     n = Node("foo", int, node_source=NodeType.EXTERNAL)
     hamilton_node = graph_types.HamiltonNode.from_node(n)
     # the above will have no specified versions
+    assert hamilton_node.version == "foo"
+    assert hamilton_node.as_dict()["version"] == "foo"
+
+
+def test_create_hamilton_node_missing_version():
+    """We contrive a case where originating_functions is None."""
+
+    def foo(i: int) -> int:
+        return i
+
+    n = Node("foo", int, "", foo, node_source=NodeType.STANDARD)
+    hamilton_node = graph_types.HamiltonNode.from_node(n)
+    # the above will have no specified versions
     assert hamilton_node.version is None
     assert hamilton_node.as_dict()["version"] is None
 
 
+def test_hamilton_graph_version_normal():
+    dr = driver.Builder().with_modules(no_parallel).build()
+    graph = graph_types.HamiltonGraph.from_graph(dr.graph)
+    assert graph.version == "3b3487599ccc4fc56995989c6d32b58a90c0b91b8c16b3f453a2793f47436831"
```

Comment:
these version values can't be hardcoded because the hashing is dependent on the Python version. see `test_create_hamilton_node()` from this file

Reply:
sure - but it seems like only python 3.8 failed.